google/cloud/aiplatform/training_jobs.py

```diff
@@ -408,6 +474,38 @@ def _create_input_data_config(
                     key=predefined_split_column_name
                 )
 
+            # Create timestamp split spec
+            timestamp_split = None
+            if timestamp_split_column_name:
+                if dataset._gca_resource.metadata_schema_uri not in (
+                    schema.dataset.metadata.tabular,
+                    schema.dataset.metadata.time_series,
+                ):
+                    raise ValueError(
+                        "A timestamp split may only be used with a tabular or time series Dataset"
+                    )
+
+                timestamp_split = gca_training_pipeline.TimestampSplit(
+                    training_fraction=training_fraction_split,
+                    validation_fraction=validation_fraction_split,
+                    test_fraction=test_fraction_split,
+                    key=timestamp_split_column_name,
+                )
+
+            fraction_split = None
```

Comment:
I agree with the comment that *_fraction_splits need to default to None throughout because we will not be able to differentiate when the user explicitly set those values and when they have been defaulted which is important as we need to determine if we received an invalid set of arguments.

If we agree on that then we need to also update the documentation that if no fraction_spit, timestamp_split, filter_split, or predfined_split is set then we default to a 0.8, 0.1, 0.1 fraction_split.

There needs to be additional logic at the end here when if all splits are None then fallback to a 0.8, 0.1, 0.1 fraction split.

I don't think this is a breaking change as we moved to setting the argument as Optional.

Reply:
everything is now none and falls back to 0.8, 0.1, 0.1. with the exception of video, where 0.8, none, 0.2 has been used.

Reply:
i'm also hesitant to have these schema specific values in the training jobs parent class.